client_end_user/shared/src/commonMain/kotlin/presentation/composable/BpImageCard.kt

```diff
@@ -85,7 +91,9 @@ fun BpImageCard(
                 color = Theme.colors.contentPrimary
             )
             Spacer(Modifier.weight(1f))
```

Comment:
try to avoid spacers and use the same modifier of the component

Reply:
it already there in card   but i think it's kay it it weight